chore(docs): removed all status tags (#7841)
swisspost/design-system: .changeset/petite-vans-cross.md

Reviewer: Could you also remove the `status:Deprecated` from the internet-header component as well as the other `status:Stable`?

Otherwise we can still filter them in here and that might be confusing:
<img width="518" height="347" alt="Image" src="https://github.com/user-attachments/assets/78af00a1-7679-466c-996e-98a530bab0e3" />

And if we're removing those status, I think it'd be best to remove the mechanism for the status icon that's in the `manager.ts` file. We can always come back to this PR if we ever want to add back status.

---
'@swisspost/design-system-documentation': patch
---

> Removed deprecated status tags "experimental" and "in progress". Components will no longer be tagged for status because it confused more than it helped.

Reviewer: ```suggestion
Removed all status tags. Components will no longer be tagged for status because it confused more than it helped.
```
